package.json

```diff
@@ -15,7 +15,7 @@
     "feature-filter": "^1.0.2",
     "geojson-vt": "^2.1.0",
     "gl-matrix": "^2.3.1",
-    "glify": "^0.5.0",
+    "glify": "^0.5.1",
```

Comment:
Yeah, this pulls in https://github.com/mapbox/glify/commit/d5a972aed8d26984c60cab73e31b4c589591b7c9 which was necessary to get glify to play nice with documentation.js's CLI.